src/client/pages/CubeDraftPage.tsx

```diff
@@ -477,22 +547,44 @@ const CubeDraftPage: React.FC<CubeDraftPageProps> = ({ cube, draft, loginCallbac
       if (target.type === locations.pack) {
         return;
       }
-
+      //Dragged a card from the pack to the deck or sideboard (the latter is off)
```

Comment:
We can remove the `(the latter is off)` from the comment now, the sideboard is available during picks